ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -385,11 +385,6 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
                     this.onPagoPAEnvironmentToggle,
                     I18n.t("profile.main.pagoPaEnvironment.pagoPAEnvAlert")
                   )}
-                {this.developerListItem(
-                  I18n.t("profile.main.debugMode"),
-                  this.props.isDebugModeEnabled,
-                  this.props.setDebugModeEnabled
```

Comment:
I guess we can remove from code _setDebugModeEnabled_ from **mapDispatchToProps** and from imports too, since it isn't used anymore